### PR TITLE
Add custom sounds

### PR DESCRIPTION
**Pull request by terabyte25**
Closes Aliucord/plugin-requests#80. Works with mp4, m4a, mp3 formats.

### CODE REVIEW

**Comment by terabyte25** [comment]
Aliucord/plugin-requests#80.

File: Themer/src/main/kotlin/dev/vendicated/aliucordplugs/themer/ThemeLoader.kt [hunk 1]
```diff
@@ -77,6 +79,15 @@ object ThemeLoader {
         }
     }
 
+    private fun loadRaw(theme: Theme, name: String, url: String) {
+        try {
+            val raw = getResourceWithCache(theme, "raw_$name", url)
+            ResourceManager.putRaw(name, AssetFileDescriptor(ParcelFileDescriptor.open(raw, ParcelFileDescriptor.MODE_READ_ONLY), 0, -1))
```
Comment (terabyte25) [inline Themer/src/main/kotlin/dev/vendicated/aliucordplugs/themer/ThemeLoader.kt:85]: don't mind the horror

File: Themer/src/main/kotlin/dev/vendicated/aliucordplugs/themer/settings/ThemerSettings.kt [hunk 2]
```diff
@@ -116,6 +116,18 @@ class ThemerSettings : SettingsPage() {
         )
         addView(Divider(ctx))
 
+        addView(Divider(ctx))
```
Comment (Vendicated) [inline Themer/src/main/kotlin/dev/vendicated/aliucordplugs/themer/settings/ThemerSettings.kt:119]: why double divider

File: Themer/src/main/kotlin/dev/vendicated/aliucordplugs/themer/Constants.kt [hunk 3]
```diff
@@ -134,6 +135,22 @@ val SIMPLE_BG_SECONDARY_ATTRS = arrayOf(
     "theme_chat_spoiler_bg"
 )
 
+val SIMPLE_SOUND_NAMES = arrayOf("call_calling",
```
Comment (Vendicated) [inline Themer/src/main/kotlin/dev/vendicated/aliucordplugs/themer/Constants.kt:138]: the first element shouldn't be in the same line

**Comment by Vendicated** [comment]
LGTM!

**Pull request merged**